cli/src/commands/run.rs

```diff
@@ -11,24 +14,30 @@ use crate::ui::Ui;
 /// # Example
 ///
 /// # Run pre-commit on your local work
-/// $ jj run 'pre-commit.py .github/pre-commit.yaml' -r (main..@) -j 4
+/// $ jj run 'pre-commit run .github/pre-commit.yaml' -r (trunk()..@) -j 4
 ///
 /// This allows pre-commit integration and other funny stuff.
 #[derive(clap::Args, Clone, Debug)]
 #[command(verbatim_doc_comment)]
 pub struct RunArgs {
     /// The command to run across all selected revisions.
+    shell_command: String,
+    /// A no-op option to match the interface of `git bisect -x`.
```

Comment:
nit: mention `git rebase -x` instead since `jj run` will be more similar to that than to `git bisect`?

Reply:
corrected